java-checks/src/test/files/checks/WriteObjectTheOnlySynchronizedMethodCheck.java

```diff
@@ -16,7 +15,6 @@ public void bounce(float angle, float velocity) {
   }
 
   private synchronized void writeObject(ObjectOutputStream stream) throws IOException { // Compliant, maybe FN
```

Comment:
why `maybe` ? if it's not considered because of the other synchronized method, from the inner class, then it should be documented for that behavior. Otherwise, just cut the exploration of classtree bodies in the `SynchronizationVisitor`.